Added my solution and a time test

carmocca opened the pull request:
My solution to #13

tonilopezmr commented:
Well @carmocca you don't have a CarlosImp because you've deleted!

 *Why?*

carmocca commented:
Sorry about that, i've fixed it now and added the files.
This pull now adds my implementation and a class called numberOfMarksCost that lets you choose which class do you want to test.

tonilopezmr commented:
only need to do one thing, the 3º point in issue #13 

In class ExamMarksFactory add your implementation CarlosImp in create() method and you will can see test passes in test package.

Use you Intellij Idea to open the project?

carmocca commented:
Hi, sorry for all these commits :sweat_smile:, ill try to be cleaner next time. Changes:

- Fixed CarlosImp, it didn't pass the tests before.
- Fixed typo Qeue.
- Added NumberOfMarksCost, a class that compares our implementations speed (It used reflection before, now it uses Factory) i also moved it from src to tests.
- Added my implementation to ExamMarksFactory.

I believe everything is working properly now. If you think everything is correct, feel free to merge.

tonilopezmr commented on java/test/problems/problem2/NumberOfMarksCost.java:
```diff
@@ -25,35 +23,34 @@
+        int [] vector;
+        double t1, t2, time;
+        t1 = t2 = time = 0;
+        List<ExamMarks> list = new ExamMarksFactory().create();
```
Wow amazing!! You can see that ExamMarksFactory is very powerful!!!

tonilopezmr commented:
No No!!! all commits is necessary!! 

It's better a lots of commits but clear and little than one big ![](http://www.aletreando.com/haiku/2012/04/caca1.png)

Thanks you guyy!! 

![](https://media.giphy.com/media/xTiTnEFPClzrN0KJvq/giphy.gif)
